soccer/gameplay/plays/stopped.py

```diff
@@ -33,6 +33,9 @@ def __init__(self):
                             lambda: not self.is_in_center(),
                             'Switched into normal mode')
 
+        self.slow_speed = 1
```

Comment:
I'm pretty sure we determined that we weren't actually moving too fast.

Reply:
Nevermind, we don't have anything preventing us from going to fast in stopped right now so we should keep this